src/gantry/highlight.py

```
@@ -0,0 +1,127 @@
+            parts.append("bold")
+        if style_dict["italic"]:
+            parts.append("italic")
+        if style_dict["underline"]:
+            parts.append("underline")
+        if style_dict["color"]:
+            parts.append(f"#{style_dict['color']}")
+
+        rich_style = " ".join(parts) if parts else "white"
+        _style_cache[token_type] = rich_style
+        return rich_style
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)